voc/logging_service.go

```diff
@@ -25,7 +27,9 @@
 
 package voc
 
+// LoggingService A logging-as-a-service offering, e.g. for analyzing logs; has a Storage resource that stores the logs
 type LoggingService struct {
 	*NetworkService
-	Storage []ResourceID `json:"storage"`
+	Storage	ResourceID	`json:"storage"`
```

Comment:
Do we intentionally want only a single storage object here? Maybe also keep the array but the "storage" name?

Reply:
Than we have to change that in the Ontology.

Reply:
Yes, ontology change. I think its reasonable to assume that a logging service has multiple locations where things are stored.